exporter/honeycombexporter/honeycomb.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2019 OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package honeycombexporter
+
+import (
+	"context"
+
+	"github.com/honeycombio/opentelemetry-exporter-go/honeycomb"
+	"github.com/open-telemetry/opentelemetry-collector/consumer/consumerdata"
+	"github.com/open-telemetry/opentelemetry-collector/exporter"
+	"github.com/open-telemetry/opentelemetry-collector/exporter/exporterhelper"
+	"github.com/open-telemetry/opentelemetry-collector/oterr"
+	"go.opentelemetry.io/otel/api/core"
+)
+
+const oTelCollectorUserAgentStr = "Honeycomb-OpenTelemetry-collector"
```

Comment:
```suggestion
const oTelCollectorUserAgentStr = "Honeycomb-OpenTelemetry-Collector"
```
?

Reply:
Good catch, changed in fa4961a732475b95b6a11705319077519d45dec8